src/cookbooks/clients_last_seen_bits.md

```diff
@@ -0,0 +1,136 @@
+```
+
+Let's make that representation a bit more visual:
+
+```
+    0  0  0  0  0  0  0  0  0  0  0  0  0  0  1  0  0  0  0  0  0  1  0  0  0  0  0  0
+    ──────────────────────────────────────────────────────────────────────────────────
+    │  │  │  │  │  │  │  │  │  │  │  │  │  │  │  │  │  │  │  │  │  │  │  │  │  │  │  │
+    │ 26  │ 24  │ 22  │ 20  │ 18  │ 16  │ 14  │ 12  │ 10  │  8  │  6  │  4  │  2  │  0
+   27    25    23    21    19    17    15    13    11     9     7     5     3     1
+   
+  └──────────────────────────────────────────────────────────────────────────────────┘
+  MAU                                                             └──────────────────┘
+                                                                   WAU             └─┘
+                                                                                   DAU
+```
+
+In this picture, we've annotated the windows for DAU (day 0 only), 
+WAU (days 0 through -6) and MAU (days 0 through -27). This particular client
+won't count toward DAU for 2020-01-28, but the client does count towards both
+WAU and MAU.
+
+You may notice that the day 13 bit doesn't end up affecting these metrics.
+For DAU, WAU, and MAU, all that matters is the rightmost bit that's set,
+telling us about the most recent activity seen for that user. We provide
+a special function for that use case:
+
+
+```
+SELECT udf.bits28_days_since_seen(8256)
+-- 6
+```
+
+Indeed, this is so commonly used that we build this function into user-facing
+views, so that instead of referencing `days_seen_bits` with a UDF, you can
+instead reference a field called `days_since_seen`. Counting MAU, WAU, and
+DAU generally looks like:
+
+```
+SELECT
+  COUNT(days_since_seen < 28) AS mau,
+  COUNT(days_since_seen <  7) AS wau,
+  COUNT(days_since_seen <  1) AS dau
+FROM
+  telemetry.clients_last_seen
+```
+
+Note that this particular client is about to fall outside the WAU window.
+If the client doesn't send a main ping on 2020-01-29, the new `days_seen_bits`
+pattern will look like:
+
+```
+    0  0  0  0  0  0  0  0  0  0  0  0  0  1  0  0  0  0  0  0  1  0  0  0  0  0  0  0
+    ──────────────────────────────────────────────────────────────────────────────────
+    │  │  │  │  │  │  │  │  │  │  │  │  │  │  │  │  │  │  │  │  │  │  │  │  │  │  │  │
+    │ 26  │ 24  │ 22  │ 20  │ 18  │ 16  │ 14  │ 12  │ 10  │  8  │  6  │  4  │  2  │  0
+   27    25    23    21    19    17    15    13    11     9     7     5     3     1
+   
+  └──────────────────────────────────────────────────────────────────────────────────┘
+  MAU                                                             └──────────────────┘
+                                                                   WAU             └─┘
+                                                                                   DAU
+```
+
+The `days_seen_bits` value is now `16512` and the user is indeed outside the WAU window:
+
+```
+SELECT udf.bits28_days_since_seen(16512)
```

Comment:
This is still a mystery to me but maybe it is ok if folks don't understand how you go from the inscrutable number to 7 days since last seen and can just rely on the UDF